parts/k8s/kubernetesparams.t

```diff
@@ -358,20 +365,23 @@
       },
       "type": "string"
     },
+{{end}}
+{{if .OrchestratorProfile.KubernetesConfig.IsAADPodIdentityEnabled}}
     "kubernetesAADPodIdentityEnabled": {
       "defaultValue": false,
       "metadata": {
         "description": "AAD Pod Identity status"
       },
       "type": "bool"
     },
+{{end}}
     "kubernetesACIConnectorEnabled": {
-      "defaultValue": false,
```

Comment:
why remove the default here?

Reply:
because it’s never relied upon in the updated implementation